File: week_1/day_3/ExercisesXP/exercise_3.js
/*
Exercise 3: Even Or Odd
Instructions
1. Prompt the user for a number and save it to a variable.
2. Check whether the variable is even or odd.
    - If it is even, display: "x is an even number". Where x is the actual number the user chose.
    - If it is odd, display: "x is an odd number". Where x is the actual number the user chose.
*/

// 1. Prompt the user for a number and save it to a variable.
let userNumber = prompt("Input a number please");
userNumber = parseInt(userNumber); // Cast the string to an integer Number type

// 2. Check whether the variable is even or odd.
if ((userNumber % 2) == 0) {
    // - If it is even, display: "x is an even number". Where x is the actual number the user chose.
    alert(`${userNumber} is an even number`)
} else if (userNumber % 2 != 0 && !(isNaN(userNumber))) {
    // - If it is odd, display: "x is an odd number". Where x is the actual number the user chose.
    alert(`${userNumber} is an odd number`)
}

File: week_2/day_2/follow_along/exercise_3.py
# Accept a number from the user and print its multiplication table

user_number = int(
    input("Please input a number, and I'll return it's multiplication table: "))
for i in range(1, 13):
    print(f"{i} X {user_number} = {i*user_number}")

# Multiplication table

for i in range(11):
   for j in range(11):
      print(i * j, end='\t')
   print()